PsfLauncher/main.cpp

```diff
@@ -35,7 +35,8 @@ using namespace std::literals;
 // Forward declarations
 void LogApplicationAndProcessesCollection();
 int launcher_main(PCWSTR args, int cmdShow) noexcept;
-void GetAndLaunchMonitor(const psf::json_object &monitor, std::filesystem::path packageRoot, int cmdShow, LPCWSTR dirStr);
+bool LaunchViaCreateProcessInContainer(const std::wstring app, const std::wstring fullCmd, const std::wstring dir, bool cmdShow);

```

Comment:
The process will always be launched in the container now.  I use the File Proc Attributes on the powershell exe only.  THis can be removed.

Reply:
Are you sure? From what I can tell only the powershell launches for SCRIPTS are fixed.  When PsfLauncher is asked to run a non exe file (originally from a shortcut pointing to a file) this is performed using shell launching in this file, and it seems to always be outside of the container.  I will remove the disabled code, but I'm thinking that we end up putting something like it back in place.  Even if we only cover a few file types like cmd/bat/ps1 we will need need some of these to run inside the container, and all of them in the container should be strived for.

Reply:
Addressed in Commit 1bdc01a3, although my previous comment still is in play.

Reply:
I see.  I'm sorry, I was confused.  You are correct.  PSF launches only scripts and powershell.exe in a container, nothing else.  

The method LaunchViaCreateProcessInContainer is for the main application that PSF will launch, right?